src/Command/CreateClassCacheCommand.php

```diff
@@ -30,8 +30,26 @@
  *
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
  */
-class CreateClassCacheCommand extends ContainerAwareCommand
+class CreateClassCacheCommand extends Command
```

Comment:
Technically a BC-break, and our classes are not final so we don't know who extends that… I have no solution besides creating new commands, so let's go with this anyway I guess :sweat_smile: